src/assets/data/edition/series/2/section/2a/m43/textcritics.json

```diff
@@ -0,0 +1,124 @@
+                            }
+                        ]
+                    }
+                ]
+            }
+        },
+        {
+            "id": "M_43_Sk1",
+            "label": "M 43 Sk1",
+            "evaluations": [
+                "<strong>Sk1</strong> ist eine rhythmische Skizze zur <a (click)=\"ref.selectSvgSheet({complexId: 'm43', sheetId: 'M_43_TF1'})\">Studienkomposition für Klavier M 43</a>."
+            ],
+            "commentary": {
+                "preamble": "",
+                "comments": [
+                    {
+                        "blockHeader": "",
+                        "blockComments": [
+                            {
+                                "svgGroupId": "g2317",
+                                "measure": "7",
+                                "system": "16",
+                                "position": "1–4/8",
+                                "comment": "Vier Achtelnoten überschreiben drei Triolenachteln."
+                            },
+                            {
+                                "svgGroupId": "g3062",
+                                "measure": "7",
+                                "system": "16",
+                                "position": "(4/4)",
+                                "comment": "Drei Triolenachteln überschreiben vier Achtelnoten."
+                            },
+                            {
+                                "svgGroupId": "g3062",
+                                "measure": "7",
+                                "system": "16",
+                                "position": "(4/4)",
+                                "comment": "Drei Triolenachteln überschreiben vier Achtelnoten."
+                            },
+                            {
+                                "svgGroupId": "g12590",
+                                "measure": "8B",
+                                "system": "17",
+                                "position": "6/8",
+                                "comment": "sic. Lies: Viertelnote."
+                            },
+                            {
+                                "svgGroupId": "g15345",
+                                "measure": "10",
+                                "system": "17",
+                                "position": "1–2/8",
+                                "comment": "Zwei Achtelnoten überschreiben drei Triolenachteln."
+                            },
+                            {
+                                "svgGroupId": "g13704",
+                                "measure": "10",
+                                "system": "17",
+                                "position": "2/4",
+                                "comment": "Viertelnote überschreibt drei Triolenachteln."
+                            }
+                        ]
```

Comment:
Triolenachtelnoten?

Reply:
Auch; TkA für g3062 ist doppelt, dafür fehlt diejenige für die Streichung in T. {8A} (= g10764).